AMM/project/scripts/swap-exact-input/src/main.sw

```diff
@@ -0,0 +1,60 @@
+script;
+
+use libraries::{AMM, Exchange};
+
+enum InputError {
+    RouteTooShort: (),
+}
+
+enum SwapError {
+    ExcessiveSlippage: u64,
+    PairExchangeNotRegistered: (ContractId, ContractId),
+}
+
+fn main(
+    amm_contract_address: ContractId,
```

Comment:
Could this be a config time constant? The user should never need to point at different AMM addresses (ironically this did happen on ethereum (naming no names), thus forever fragmenting liquidity. But I hope that's an exception rather than a rule)

Reply:
Done! Hardcoded for now since as far as I know we cannot set config-time constants via the SDK. But the AMM should not be updated too often that this becomes a chore until they can be set programatically.